src/main/java/com/example/cabbooking/controller/CabBookingController.java

```diff
@@ -47,12 +48,12 @@ public ResponseEntity<String> addDriver(@RequestBody DriverDetailsVO driverDetai
     }
 
     @GetMapping("/find_ride")
```

Comment:
please read this and use proper naming convension for API endpoints 
https://pratapsharma.io/api-naming-convention

Reply:
Will taken care and now ressolved.